app/routes/home/components/weatherWidget/weatherWidget.container.js

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { createStructuredSelector } from 'reselect';
+
+import { WeatherWidget } from './weatherWidget.component';
+import { WeatherActions } from '../../../../modules/weather/weather.redux';
+import { selectWeather } from '../../../../modules/weather/weather.selectors';
+
+
+const mapStateToProps = createStructuredSelector({
+  weather: selectWeather(),
```

Comment:
jeśli selector nie przyjmuje parametrów to można przy implementacji zrezygnować z funkcji wrapującej i wtedy tylko się używa `selectWeather`